sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryId.kt

```diff
@@ -0,0 +1,35 @@
+package io.sentry.kotlin.multiplatform
+
+class SentryId(private val sentryIdString: String) {
```

Comment:
Can't we use the SentryId from the Java SDK here? or because it's java it's not possible?